component/Avatar/avatar.css

```diff
@@ -0,0 +1,50 @@
+/* avatar css */
+.avatar-container{
+    margin: auto;
+    width: 500px;
+}
+
+.box{
+    height:100px;
```

Comment:
use rem units everywhere to achieve responsiveness for the future. pixels will cause you trouble

Reply:
done